Sources/AppcuesKit/Data/Models/Experiment.swift

```diff
@@ -12,6 +12,8 @@ internal struct Experiment {
     let group: String
     let experimentID: UUID
     let experienceID: UUID
+    let goalID: String
```

Comment:
per Nate, this needs to be a string, cannot be assumed a UUID due to legacy goal usage in Firebase